platforms/dji/tello/driver.go

```diff
@@ -195,9 +195,9 @@ type Driver struct {
 
 // NewDriver creates a driver for the Tello drone. Pass in the UDP port to use for the responses
 // from the drone.
-func NewDriver(port string) *Driver {
+func NewDriver(ip string, port string) *Driver {
```

Comment:
The only issue here is that this is a breaking API change.

Perhaps a better way to achive this is to add another function `NewDriverWithIP(ip string, port string)` so that existing code continues to work, but that this functionality can be provided.

What do you think?

Reply:
I was worried about the same thing, let me have a go at it

Reply:
fixed